### PR TITLE
CT-323 Closed Case Page

### PR DESCRIPTION
**Pull request by stephenrichards**

### CODE REVIEW

File: app/views/cases/closed_cases.html.slim [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+              = " "
+              = link_to kase.number, case_path(kase.id)
+            td aria-label="#{t('.name-subject')}"
+              <strong>
+              = kase.name
+              </strong>
+              <br>
+              <span>
+              = kase.subject
+              </span>
```
Comment (aliuk2012) [inline app/views/cases/closed_cases.html.slim]: Lines 24 to 30 should be in slim not HTML

File: config/locales/en.yml [hunk 2]
```diff
@@ -208,6 +208,11 @@ en:
       case_created: Case successfully created
 
   cases:
+    closed_cases:
+      page_heading: Closed Cases
```
Comment (aliuk2012) [inline config/locales/en.yml]: Lowercase 'c' for cases

File: config/locales/en.yml [hunk 3]
```diff
@@ -258,6 +263,7 @@ en:
 
   nav:
     cases: Cases
+    closed_cases: Closed Cases
```
Comment (aliuk2012) [inline config/locales/en.yml]: Lowercase 'c'

File: config/locales/en.yml [hunk 4]
```diff
@@ -258,6 +263,7 @@ en:
 
   nav:
     cases: Cases
```
Comment (aliuk2012) [inline config/locales/en.yml]: Shouldn't this change to "Open cases"?

File: app/views/cases/closed_cases.html.slim [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+        - @cases.each do |kase|
+          tr.case_row
+            td aria-label="#{t('.number')}"
+              span.visually-hidden
+                = t('.view_case')
+              = " "
+              = link_to kase.number, case_path(kase.id)
+            td aria-label="#{t('.name-subject')}"
+              <strong>
+              = kase.name
```
Comment (aliuk2012) [inline app/views/cases/closed_cases.html.slim]: should be tabbed

File: app/views/cases/closed_cases.html.slim [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+              </strong>
+              <br>
+              <span>
+              = kase.subject
```
Comment (aliuk2012) [inline app/views/cases/closed_cases.html.slim]: should be tabbed

File: config/locales/en.yml (hunk 2 above)
Comment (stephenrichards) [inline config/locales/en.yml]: correct

File: app/views/cases/closed_cases.html.slim (hunk 1 above)
Comment (stephenrichards) [inline app/views/cases/closed_cases.html.slim]: correct

File: config/locales/en.yml (hunk 4 above)
Comment (stephenrichards) [inline config/locales/en.yml]: correct

File: config/locales/en.yml (hunk 3 above)
Comment (stephenrichards) [inline config/locales/en.yml]: yep

File: app/views/cases/closed_cases.html.slim (hunk 5 above)
Comment (stephenrichards) [inline app/views/cases/closed_cases.html.slim]: Addressed by converting to slim

File: app/views/cases/closed_cases.html.slim (hunk 6 above)
Comment (stephenrichards) [inline app/views/cases/closed_cases.html.slim]: Addressed by converting to slim

**Comment by stephenrichards** [comment]
I've written a view spec for the Global Nav Partial.  I think this, along with GlobalNavManager spec means that we can do away with most of the Primary Navigation feature spec, retaining the bits that check that it's not visible at all on the login page.

In the Global Nav spec we test that the correct nav links are set up depending on role (or will do in the future), and in the view spec, we test that the right links are produced given a particular GlobalNavManager set up, and that the link is marked as active if the current_page?(url) returns true.

That should be enough, I think; we're testing the mechanism, and therefore don't have to test each and every possible combination.

Let me know what you think.

**Comment by misaka** [comment]
@stephenrichards Agreed that we can start to replace bloated feature tests that test every single and detail of the views with view specs, and focus primarily on user journeys in the feature tests.

**Pull request merged**